### PR TITLE
 Account UI create

### PR DESCRIPTION
**Pull request by rukshand2001**

### CODE REVIEW

File: src/main/java/org/example/BankingApp.java [hunk 1]
```diff
@@ -69,6 +69,9 @@ public void createYourAccountUI(Scanner scanner){
         newAccount.setInterestRate(BankingApp.INTEREST_RATE);
 
         BankingApp.accounts.add(newAccount);
+        System.out.println("Your account number is: "+accountNumber);
+
+        this.loggingUI(scanner);
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]: we don't need to add this

File: src/main/java/org/example/BankingApp.java [hunk 2]
```diff
@@ -96,4 +99,51 @@ public Account getAccountByID(int accountNumber){
         }
         return null;
     }
+
+    public void bankingAppAccountUI(int accountNumber){
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]: pass scanner as a parameter here

File: src/main/java/org/example/BankingApp.java [hunk 3]
```diff
@@ -96,4 +99,51 @@ public Account getAccountByID(int accountNumber){
...
+        /*to do banking works with accounts*/
+
+        Scanner scanner = new Scanner(System.in);
+        boolean loggedIn = true;
+
+        while (loggedIn){
+            System.out.println("Press 1 to withdrawal money");
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
            System.out.println("Press 1 to withdraw money");
```

File: src/main/java/org/example/BankingApp.java [hunk 4]
```diff
@@ -96,4 +99,51 @@ public Account getAccountByID(int accountNumber){
...
+            System.out.println("Press 2 to deposit money");
+            System.out.println("press 3 to get balance");
+            System.out.println("pres 4 to get your account interest");
+            System.out.println("press 5 to do transactions");
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
            System.out.println("press 5 to do a transaction");
```

File: src/main/java/org/example/BankingApp.java [hunk 5]
```diff
@@ -96,4 +99,51 @@ public Account getAccountByID(int accountNumber){
...
+            System.out.println("press 6 to exit");
+
+            System.out.print("Enter your choose: ");
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
            System.out.print("Enter your choice: ");
```

File: src/main/java/org/example/BankingApp.java [hunk 6]
```diff
@@ -96,4 +99,51 @@ public Account getAccountByID(int accountNumber){
...
+            int choose =scanner.nextInt();
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
            int choice =scanner.nextInt();
```

File: src/main/java/org/example/BankingApp.java [hunk 7]
```diff
@@ -96,4 +99,51 @@ public Account getAccountByID(int accountNumber){
...
+                    System.out.print("Your currently interest is: ");
+                    break;
+                case 5:
+                    System.out.print("Receiver's  account number & amount: ");
+                    break;
+                case 6:
+                    loggedIn = false;
+                    break;
+                default:
+                    System.out.println("Enter valued choose");
```
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
                    System.out.println("You have entered a wrong choice. Please check your choice");
```

File: src/main/java/org/example/BankingApp.java (hunk 2 above)
Comment (aselarbd) [inline src/main/java/org/example/BankingApp.java]:
```suggestion
    public void accountLoggedUI(int accountNumber){
```

File: src/main/java/org/example/BankingApp.java (hunk 1 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: this means print?

Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: done

File: src/main/java/org/example/BankingApp.java (hunk 2 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: Screenshot (100)
 I also adde sacnner  here

Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: done

File: src/main/java/org/example/BankingApp.java (hunk 3 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: corrected

File: src/main/java/org/example/BankingApp.java (hunk 4 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: corrected

File: src/main/java/org/example/BankingApp.java (hunk 5 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: corrected

File: src/main/java/org/example/BankingApp.java (hunk 6 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: corrected

File: src/main/java/org/example/BankingApp.java (hunk 7 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: corrected

File: src/main/java/org/example/BankingApp.java (hunk 2 above)
Comment (rukshand2001) [reply src/main/java/org/example/BankingApp.java]: corrected

**Review by aselarbd — APPROVED** [review]

**Pull request merged**